examples/official-storybook/stories/core/layout.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import { Button } from '@storybook/react/demo';
+
+export default {
+  title: 'Core/Layout MDX',
+};
+
+export const Default = () => <Box>padded by default</Box>;
```

Comment:
All these functions need to be in a CSF file @ndelangen

Reply:
I deleted all those exports, they aren't needed.